src/test/java/org/fundacionjala/coding/carmen/SortContentTest.java

```diff
@@ -0,0 +1,25 @@
+package org.fundacionjala.coding.carmen;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+public class SortContentTest {
```

Comment:
Your implementation is failing with this scenario 
```java
assertEquals("I am", SortContent.sortTheInnerContent("I am"));
```

Reply:
thanks, observation resolved